rest-api/src/main/scala/za/co/absa/enceladus/rest_api/SpringFoxConfig.scala

```diff
@@ -28,27 +28,55 @@ import za.co.absa.enceladus.utils.general.ProjectMetadata
 @Configuration
 @EnableSwagger2
 class SpringFoxConfig extends ProjectMetadata {
+
+  import org.springframework.beans.factory.annotation.Value
+
+  @Value("${spring.profiles.active:}")
+  private val activeProfiles: String = null
+
   @Bean
   def api(): Docket = {
+    val isDev = activeProfiles.split(",").map(_.toLowerCase).contains("dev")
+
     new Docket(DocumentationType.SWAGGER_2)
-      .apiInfo(apiInfo)
+      .apiInfo(apiInfo(isDev))
       .select
       .apis(RequestHandlerSelectors.any)
-      .paths(filteredPaths)
+      .paths(filteredPaths(isDev))
       .build
   }
 
-  private def filteredPaths: Predicate[String] =
-    or[String](regex("/api/dataset.*"), regex("/api/schema.*"),
+  private def filteredPaths(isDev: Boolean): Predicate[String] = {
+    val v2devPaths = Seq(
+      regex("/api/dataset.*"), regex("/api/schema.*"),
       regex("/api/mappingTable.*"), regex("/api/properties.*"),
-      regex("/api/monitoring.*"),regex("/api/runs.*"),
+      regex("/api/monitoring.*"), regex("/api/runs.*"),
       regex("/api/user.*"), regex("/api/spark.*"),
       regex("/api/configuration.*")
     )
+    val v2prodPaths = Seq(
+      regex("/api/.*/importItem"),
+      regex("/api/.*/exportItem/.*")
+    )
+    val v3paths = Seq(
+      regex("/api-v3/datasets.*"), regex("/api-v3/schemas.*"),
+      regex("/api-v3/mapping-tables.*"), regex("/api-v3/property-definitions.*")
+    )
+
+    val paths: Seq[Predicate[String]] = if (isDev) {
+      v2devPaths ++ v3paths
+    } else {
+      v2prodPaths ++ v3paths
```

Comment:
```suggestion
      v3paths
```

So prod APIs should be only v3 paths as the import-export being "prod" is only an ABSA thing. 

This would result in dev having all v2 and v3 paths and prod only v3.

Reply:
Thanks for the clarification, suggestion applied (along with removing the unnecessary code)